lib/generators/react_on_rails/base_generator.rb

```diff
@@ -85,9 +85,9 @@ def update_application_js
 
       def strip_application_js_of_incompatible_sprockets_statements
         application_js = File.join(destination_root, "app/assets/javascripts/application.js")
-        gsub_file(application_js, "//= require jquery_ujs", "")
-        gsub_file(application_js, %r{//= require jquery$}, "")
-        gsub_file(application_js, "//= require_tree .", "")
+        gsub_file(application_js, "//= require jquery_ujs", "// require jquery_ujs")
+        gsub_file(application_js, %r{//= require jquery$}, "// require jquery")
+        gsub_file(application_js, %r{//= require_tree .$}, "// require_tree .")
```

Comment:
Yeah, you right... But it covers the case correctly here when there is a `$` in the ending. Anyway, it's better to code with preciseness. I will fix it.